### PR TITLE
Bai 13

### PR DESCRIPTION
**Pull request by TienDatNguyen321**
You are choreographing a circus show with various animals. For one act, you are given two kangaroos on a number line ready to jump in the positive direction (i.e, toward positive infinity).

The first kangaroo starts at location x1 and moves at a rate of v1 meters per jump.
The second kangaroo starts at location x2 and moves at a rate of v2 meters per jump.
You have to figure out a way to get both kangaroos at the same location at the same time as part of the show. If it is possible, return YES, otherwise return NO.

Function Description

Complete the function kangaroo in the editor below.

kangaroo has the following parameter(s):

int x1, int v1: starting position and jump distance for kangaroo 1
int x2, int v2: starting position and jump distance for kangaroo 2
Returns

string: either YES or NO
Input Format

A single line of four space-separated integers denoting the respective values of x1, v1,x2 , and v2 .

### CODE REVIEW

File: 2FirstWeeks/NumberLineJumps/Program.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+                    var numberOfJumps = ((double)(x1 - x2)) / (v2 - v1);
+
+                    if (numberOfJumps % 1 == 0)
+                        a = "YES";
+                    else
+                    {
+                        a = "NO";
+                    }
+
```
Comment (mvn-dungdang-hn) [inline 2FirstWeeks/NumberLineJumps/Program.cs:32]:
```suggestion
```

File: 2FirstWeeks/NumberLineJumps/Program.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            {
+                string a;
+                if ((x1 < x2 && v1 < v2) || (x2 < x1 && v2 < v1))//ở xa hơn,tốc độ kém hơn => k thể kịp
+                {
+                    return "NO";
+                }
+                //ở xa hơn nhưng chỉ cần tốc độ cao hơn => chắc chắn sẽ kịp
+                else if (x2 < x1)
+                {
+
```
Comment (mvn-dungdang-hn) [inline 2FirstWeeks/NumberLineJumps/Program.cs:23]:
```suggestion
```

File: 2FirstWeeks/NumberLineJumps/Program.cs [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+using System.Threading.Tasks;
+
+namespace NumberLineJumps
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+              string Kangaroo(long x1, long v1, long x2, long v2)
+            {
```
Comment (mvn-dungdang-hn) [inline 2FirstWeeks/NumberLineJumps/Program.cs:14]: indent chưa đúng

File: 2FirstWeeks/NumberLineJumps/Program.cs [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                    //v1 > v2
+                    var numberOfJumps = ((double)(x2 - x1)) / (v1 - v2);
+                    //check whether number of jumps is a whole number  i.e no fractional part.
+                    if (numberOfJumps % 1 == 0)
+                        a = "YES";
+                    else
+                    {
+                        a = "NO";
+                    }
+
```
Comment (mvn-dungdang-hn) [inline 2FirstWeeks/NumberLineJumps/Program.cs:45]:
```suggestion
```

File: 2FirstWeeks/NumberLineJumps/Program.cs [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+                }
+                else
+                {
+                    //v1 > v2
+                    var numberOfJumps = ((double)(x2 - x1)) / (v1 - v2);
```
Comment (mvn-dungdang-hn) [inline 2FirstWeeks/NumberLineJumps/Program.cs:37]: sao chỗ này k dùng abs thì đỡ phải else 1 phát nữa ở bên dưới

Comment (TienDatNguyen321) [reply 2FirstWeeks/NumberLineJumps/Program.cs:37]: dạ cái này đang check xem nó có nhanh hơn k mà anh :v vì đề bài bảo p trả ra yes hoặc no ạ

**Pull request merged**